constants/languages/turkish.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
You need to use it in the [`app.json`](https://github.com/ivgraai/expiry-util/blob/c220d2077743e781d02d2c364d28100f60985eb0/app.json#L33), please check [this](https://docs.expo.io/distribution/app-stores/?redirected#localizing-your-ios-app).